packages/astro/src/stores/external.ts

```diff
@@ -124,3 +121,19 @@ export const $signInStore = computed([$clientStore], client => client?.signIn);
  * $signUpStore.subscribe((signUp) => console.log(signUp.status))
  */
 export const $signUpStore = computed([$clientStore], client => client?.signUp);
+
+/**
+ * Records a telemetry event when a store is used to match React hooks telemetry.
+ *
+ * @param {Store} store - The nanostore instance to monitor.
+ * @param {string} method - The name of the method associated with the store usage.
+ */
+const recordTelemetryEvent = (store: Store, method: string) => {
+  onMount(store, () => {
+    $clerk.get()?.telemetry?.record(eventMethodCalled(method));
+  });
+};
+
+recordTelemetryEvent($signInStore, '$signInStore');
+recordTelemetryEvent($signUpStore, '$signUpStore');
+recordTelemetryEvent($organizationStore, '$organizationStore');
```

Comment:
❓ Shall we add do this for all stores ?

Reply:
The idea is to match the recording of the React hooks which only records `useSignIn`, `useSignUp`, `useOrganization` and `useOrganizationList`. But if it makes sense to record all of the stores, I'm not sure 🤔

Reply:
What if we just do it to all stores in this file, and if it gets too crowded in the analytics later, update it to only log which makes sense?

Reply:
As discussed offline, we will open a new PR if we need to add telemetry to the remaining stores